### ISSUE TITLE
Is this project active?

### ISSUE THREAD

**Issue by codehangen** [Svetloslav15/react-sidebar-ui#34]
Hi all.

I want to help, but I want to know if this project is active. *Is this project active?*

**Comment by Svetloslav15** [comment]
Yes, it didn't have active development in the recent time, but I am starting to extend the library with new cool features so if you would like to take part you are welcome. New major version 2.0.0 is going to be released soon.
